mappings/net/minecraft/command/selector/TargetSelectorParser.mapping

```diff
@@ -0,0 +1,131 @@
+CLASS p net/minecraft/command/selector/TargetSelectorParser
+	CLASS p$1
+		METHOD a apply (Lsg;)Z
+			ARG 0 entity
+	CLASS p$2
+		METHOD a apply (Lsg;)Z
+			ARG 0 entity
+	CLASS p$3
+		METHOD a apply (Lsg;)Z
+			ARG 0 entity
+	CLASS p$4
+		METHOD a apply (Lsg;)Z
+			ARG 0 entity
+	CLASS p$5
+		METHOD a compare (Lsg;Lsg;)I
+			ARG 0 o1
+			ARG 1 o2
+	CLASS p$6
+		METHOD a apply (Lsg;)Z
+			ARG 0 entity
+	CLASS p$7
+		METHOD a apply (Lsg;)Z
+			ARG 0 entity
+	CLASS p$8
+		METHOD a apply (Lsg;)Z
+			ARG 0 entity
+	CLASS p$9
+		METHOD a apply (Lsg;)Z
+			ARG 0 entity
+	CLASS p$10
+		METHOD a apply (Lsg;)Z
+			ARG 0 entity
+	CLASS p$11
+		METHOD a apply (Lsg;)Z
+			ARG 0 entity
+	CLASS p$12
+		METHOD a apply (Lsg;)Z
+			ARG 0 entity
+	CLASS p$13
+		METHOD a apply (Lsg;)Z
+			ARG 0 entity
+	FIELD a TARGET_SELECTOR_PATTERN Ljava/util/regex/Pattern;
+	FIELD b SPLIT_COMMA Lcom/google/common/base/Splitter;
+	FIELD c SPLIT_EQUALS_PAIR Lcom/google/common/base/Splitter;
+	FIELD d POSITIONAL_ARGUMENTS Ljava/util/Set;
+	METHOD a createBoundingBox (Lcn;III)Lbcs;
+		ARG 0 pos
+		ARG 1 xOffset
+		ARG 2 yOffset
+		ARG 3 zOffset
+	METHOD a canReturnMultiple (Ljava/lang/String;)Z
+		ARG 0 selector
+	METHOD a sortAndLimit (Ljava/util/List;Ljava/util/Map;Ln;Ljava/lang/Class;Ljava/lang/String;Lbcv;)Ljava/util/List;
+		ARG 0 targets
+		ARG 1 rawArguments
+		ARG 2 source
+		ARG 3 clazz
+		ARG 4 variable
+		ARG 5 origin
+	METHOD a extractScoreArguments (Ljava/util/Map;)Ljava/util/Map;
+		ARG 0 rawArguments
+	METHOD a extractRadiusPredicate (Ljava/util/Map;Lbcv;)Ljava/util/List;
+		ARG 0 rawArguments
+		ARG 1 origin
+	METHOD a extractCoordinate (Ljava/util/Map;Lcn;)Lcn;
+		ARG 0 rawArguments
+		ARG 1 defaultPos
+	METHOD a getTargetsInWorld (Ljava/util/Map;Ljava/lang/Class;Ljava/util/List;Ljava/lang/String;Laiw;Lcn;)Ljava/util/List;
```

Comment:
This is but one step to the process, and if called directly may cause unexpected partial parses.